indica/readers/ppfreader.py

```diff
@@ -255,7 +255,11 @@ def _get_charge_exchange(
         results["R"] = R.data[0, :]
         results["z"] = z.data[0, :]
         results["length"] = R.data.shape[1]
-        results["element"] = ELEMENTS_BY_MASS[int(round(mass.data[0]))]
+        results["element"] = [
+            value[2]
+            for value in ELEMENTS.values()
+            if value[1] == int(round(mass.data[0]))
+        ][0]
```

Comment:
Not an issue with this change but isn't getting element name by mass dangerous since mass is non-unique?
CX also defines a dtype "ZQNN" that gives the Z number of the main impurity, could use that to select element? (either instead or as well as mass, since Z and mass will be unique even with H isotopes)

Reply:
Yeah, I think this is a remnant of the old dictionaries where in ELEMENTS_BY_MASS, "calcium" and "cobalt" were commented out for what I suspect is this exact reason. Using Z-number and mass together should return a unique element, I'll add that in. Would the atomic number collection just be added to the same place where mass is recovered ~L250 and in the same fashion as well? I can't really test so I'll paste the line I want to add here and you can have a look to see if it's right, it would save me re-commiting and having to go through the CI again:
`atomic_num, atomic_num_path = self._get_signal(uid, instrument, "zqnn", revision)`
The new condition for checking would now be:
`if value[0] == atomic_num.data[0] and value[1] == int(round(mass.data[0]))`

Reply:
Firstly I'm being overkill suggesting "ZQNN" since I forgot about dtype "Z", which makes more sense when we just want Z...

Only thing that needs changing is to check against `int(atomic_num.data[0])` because for some reason it's returned as a float (don't know if that's a PPF or SAL issue though), otherwise I just tested it and it looks good